modules/Bio/EnsEMBL/Variation/Pipeline/SpliceAI/SplitFiles.pm

```diff
@@ -0,0 +1,235 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+
+=head1 CONTACT
+
+Please email comments or questions to the public Ensembl
+developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+Questions may also be sent to the Ensembl help desk at
+<http://www.ensembl.org/Help/Contact>.
+
+=cut
+package Bio::EnsEMBL::Variation::Pipeline::SpliceAI::SplitFiles;
+
+use strict;
+use warnings;
+use base ('Bio::EnsEMBL::Variation::Pipeline::SpliceAI::BaseSpliceAI');
+
+use FileHandle;
+use Bio::EnsEMBL::IO::Parser::BedTabix;
+use Bio::EnsEMBL::IO::Parser::VCF4Tabix;
+
+sub run {
+  my $self = shift;
+  $self->set_chr_from_filename();
+  # check = 0 (don't check transcripts); check = 1 (check transcripts)
+  my $check = $self->param_required('check_transcripts');
+
+  # check new Mane transcripts
+  if($check){
+    $self->get_new_transcripts();
+    $self->check_split_vcf_file();
+  }
+  $self->split_vcf_file();
+}
+
+sub split_vcf_file {
+  my $self = shift;
+  my $check = $self->param_required('check_transcripts');
+  my $vcf_file = $self->param_required('vcf_file');
+  my $tmp_split_vcf_dir = $self->param_required('tmp_split_vcf_dir');
+  my $input_dir = $check ? $self->param('input_dir_subset') : $self->param_required('input_directory');
```

Comment:
Where is input_dir_subset coming from?

Reply:
It's created in check_split_vcf_file L157